docs/Testing.md

```diff
@@ -49,11 +49,14 @@ Testing with dynamic tensor sizes is most easily performed by using the followin
 cmake --build . --config Release --target check-onnx-backend-dynamic[-jni]
 ``` 
 
-The onnx node tests usually have known dimension size for input tensors. So, to test tensor with unknown dimension, the model importer (Build/FrontendONNXTransformer.cpp) provides a functionality to generate such cases. When the environment variable, `IMPORTER_FORCE_DYNAMIC`, is set, the frontend import will turn the all the dimensions (by default) of all the input tensors of the model into -1. For example,
+The onnx node tests usually have known dimension size for input tensors. So, to test tensor with unknown dimension, the model importer (Build/FrontendONNXTransformer.cpp) provides a functionality to generate such cases. When the environment variable, `IMPORTER_FORCE_DYNAMIC`, is set, the frontend import will turn the all the dimensions (by default) of all the input tensors of the model into -1. As default, we assume that first dimension of all the inputs are the same, and the second dimension of all inputs are the same, and so on.
+
+For example,
 ```
 IMPORTER_FORCE_DYNAMIC='-1:-1' all dimensions of all the inputs will be changed
 IMPORTER_FORCE_DYNAMIC='0:-1' all dimensions of the first input will be changed
 IMPORTER_FORCE_DYNAMIC='0:-1|1:0,1' all dimensions of the first input and the 1st and 2nd dimensions of the second input will be changed
+IMPORTER_FORCE_DYNAMIC='0:0:a,1:b,2:c|1:0:a,1:b,2:c' the first three dimensions of the first of and the second inputs are changed. And assume that the first dimensions of the first and second arguments are the same, and same for the the second and third dimensions.
```

Comment:
I know by using `:` we can pass `dimIndex` directly to an argument attribute, but I recommend to use a different symbol rather than `:` for `dimIndex`, because `:` has already been used to separate `inputIndex` and `dimString`. It confuses the parsers. How about `-` or `=`?

This example is quite trivial. Can we shuffle the symbols a bit, e.g. `0:0=a,1=b,2=c|1:0,1=c,2=b`, and we can have a mix of `<index>` and `<index>  '=' <symbol>`

Reply:
@tungld I changed the notation according to this comment. Thanks for the suggestions!